src/reducers/reducers.test.js

```diff
@@ -1,15 +1,75 @@
 import { books } from './books';
 import { haveRead } from './haveRead';
-import { list } from './list';
+import { listName } from './list';
```

Comment:
Great job getting this started, I'm sure testing haveRead & toRead will be easy now that you've fixed the bug